### PR TITLE
Update the google hangout for SIG-Azure and Update SIG Index

### PR DESCRIPTION
**Pull request by emagana**

### CODE REVIEW

**Review by ezimanyi — APPROVED** [review]

**Comment by ezimanyi** [comment]
Merging after only one approval, as it's just updating links.

**Comment by emagana** [comment]
Thanks!

File: sig-index.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Don't see a SIG here that fits what you're looking for? [Submit a Proposal](sig-
 * [Spinnaker-as-Code](sig-spinnaker-as-code/README.md)
 * [Summit](sig-summit/README.md)
 * [UI / UX](sig-ui-ux/README.md)
+* [Azure](sig-azure/README.md)
```
Comment (plumpy) [inline sig-index.md:17]: These are all alphabetically sorted, so could you move it up below AWS?

Comment (emagana) [reply sig-index.md:17]: Oh, totally.. I did miss it.. New PR coming!

**Pull request merged**